### PR TITLE
Create TypeScript wrappers for the ExternalSource-related classes and…

### PR DESCRIPTION
**Pull request by scsewall**
… relationships introduced in BisCore 1.0.13

### CODE REVIEW

**Comment by scsewall** [comment]
Cannot merge until BisCore 1.0.13 is released.

File: core/backend/src/test/standalone/ExternalSource.test.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  Code, ExternalSourceAttachmentProps, ExternalSourceProps, IModel, RepositoryLinkProps, SynchronizationConfigLinkProps,
+} from "@bentley/imodeljs-common";
+import {
+  BackendRequestContext, ExternalSource, ExternalSourceAttachment, ExternalSourceAttachmentAttachesSource, ExternalSourceGroup,
+  ExternalSourceGroupGroupsSources, ExternalSourceIsInRepository, ExternalSourceOwnsAttachments, IModelDb, LinkElement, RepositoryLink, SnapshotDb,
+  SynchronizationConfigLink, SynchronizationConfigProcessesSources, SynchronizationConfigSpecifiesRootSources,
+} from "../../imodeljs-backend";
+import { IModelTestUtils } from "../IModelTestUtils";
+
+describe.only("ExternalSource", () => {
```
Comment (scsewall) [inline core/backend/src/test/standalone/ExternalSource.test.ts:18]: Remove `.only` when ready to merge

**Comment by pmconne** [comment]
1.0.13 will include RenderTimeline too, right? I have a branch where I have added the TypeScript side of that, which will follow this PR.

**Comment by scsewall** [comment]
> 1.0.13 will include RenderTimeline too, right? I have a branch where I have added the TypeScript side of that, which will follow this PR.

Correct. Here is the [BisCore 1.0.13 PR](https://dev.azure.com/bentleycs/iModelTechnologies/_git/bis-schemas/pullrequest/139608) that must complete first, and then an `imodel02` build must consume it, and then an `imodeljs` build must consume a new addon.

**Comment by scsewall** [comment]
@swwilson-bsi Could you please review?

**Comment by scsewall** [comment]
Need to add missing `toJSON` methods.

**Review by swwilson-bsi — APPROVED** [review]

**Pull request merged**